src/flashbar/__tests__/runtime-content.test.tsx

```diff
@@ -0,0 +1,305 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import * as React from 'react';
+import { act, render } from '@testing-library/react';
+
+import Button from '../../../lib/components/button';
+import Flashbar from '../../../lib/components/flashbar';
+import awsuiPlugins from '../../../lib/components/internal/plugins';
+import { awsuiPluginsInternal } from '../../../lib/components/internal/plugins/api';
+import { AlertFlashContentConfig } from '../../../lib/components/internal/plugins/controllers/alert-flash-content';
+import FlashbarWrapper from '../../../lib/components/test-utils/dom/flashbar';
+import FlashWrapper from '../../../lib/components/test-utils/dom/flashbar/flash';
+
+import stylesCss from '../../../lib/components/flashbar/styles.css.js';
+
+const pause = (timeout: number) => new Promise(resolve => setTimeout(resolve, timeout));
+
+const expectFlashContent = (
```

Comment:
Any chance we can reuse test code between alert and flashbar?

For example, many components now have analytics metadata and there are some utils to reduce boilerplate, e.g. the validateComponentNameAndLabels.